src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -164,6 +164,10 @@ public DurableTaskExtension()
             : this(options, loggerFactory, nameResolver, orchestrationServiceFactory, shutdownNotification, durableHttpMessageHandlerFactory)
         {
             this.connectionStringResolver = connectionStringResolver;
+
+            var messageSerializerSettingsFactory = new MessageSerializerSettingsFactory();

```

Comment:
I'm not convinced that this is the place that will actually fix the issue. It looks like this constructor calls into the other constructor that already initializes these settings. In fact, I think this is just a legacy constructor that may no longer be needed for Functions V1 to function.

I would test this manually to see if this works, and if it doesn't consider moving this into `InitializeForFunctionsV1()`